scalding-serialization/src/main/scala/com/twitter/scalding/serialization/macros/impl/ordered_serialization/providers/ImplicitOrderedBuf.scala

```diff
@@ -27,13 +27,11 @@ import com.twitter.scalding.serialization.macros.impl.ordered_serialization._
   type. This is for the case where its an opaque class to our macros where we can't figure out the fields
 */
 object ImplicitOrderedBuf {
-  val macroMarker = "MACROASKEDORDEREDSER"
```

Comment:
It was being used to block infinite recursion on the implicit. So instead of asking for type X, you as for the alias, and then the macros could see you were looking for an alias and would nop out of it. But if we just only use the implicit code paths on inner fields it becomes unnecessary.